fslib/ui/ui_filepicker.ui

```diff
@@ -21,23 +21,137 @@
   </property>
   <property name="windowIcon">
    <iconset>
-    <normaloff>../../fsfp/Menu/fsfp.ico</normaloff>../../fsfp/Menu/fsfp.ico</iconset>
+    <normaloff>../../../../../fsfp/Menu/fsfp.ico</normaloff>../../../../../fsfp/Menu/fsfp.ico</iconset>
```

Comment:
Considering that `../../fsfp` points to the `fsfp` directory in the root of this repository this change seems odd, since it would point outside of the repository.